src/lib/utilities/deleteAllChats.ts

```diff
@@ -0,0 +1,77 @@
+export const deleteAllChats = (): Error | void => {
+    try {
+        const chatHistory = document.querySelector(
```

Comment:
add an inline comment here that this querySelector may need to be updated if the domain updates their html